### DISCUSSION TITLE
Vue devtool marketplace

### DISCUSSION THREAD

**Discussion by Mikaleb** [vuejs/devtools-v6#1904] [Q&A] [answered]
### What problem does this feature solve?
Hi,

I see that we can add plugins to vue devtools, but there is no listing of all available plugins (a bit like vscode).

Could we see this in a near future ? Otherwise the incentive to create a new plugin is not enough.

### What does the proposed API look like?
In the vue devtool, instead of telling us how to create a plugin, tell us where to find them on a marketplace and install them easily

**Comment by Akryum** [comment, marked as answer]
Vue devtools plugins are embedded in Vue plugins, such as `vue-router`, `pinia`...
